TicTacToe.Test/BoardTest.cs

```diff
@@ -119,39 +124,41 @@ public void CheckForActualDraw()
         public void FalseDrawIfWin()
         {
             Board board = new Board(3);
-            // x o x
-            // x x o
-            // o x x
-            board.GameBoard[0] = 'x';
-            board.GameBoard[1] = 'o';
-            board.GameBoard[2] = 'x';
-            board.GameBoard[3] = 'x';
-            board.GameBoard[4] = 'x';
-            board.GameBoard[5] = 'o';
-            board.GameBoard[6] = 'o';
-            board.GameBoard[7] = 'x';
-            board.GameBoard[8] = 'x';
+            Helpers helper = new Helpers();
+            int[] positions = { 0, 1, 2, 3, 4, 5, 6, 7, 8 };
+            char[] symbols = { 'x', 'o', 'x', 'x', 'x', 'o', 'o', 'x', 'x' };
+            helper.FillBoard(board, positions, symbols);
             bool actual = board.CheckDraw('x', 'o');
             Assert.False(actual);
         }
 
+
         [Fact]
         public void FalseDrawIfBoardHasAvailableMoves()
         {
             Board board = new Board(3);
-            // - o x
-            // x x o
-            // o x x
-            board.GameBoard[1] = 'o';
-            board.GameBoard[2] = 'x';
-            board.GameBoard[3] = 'x';
-            board.GameBoard[4] = 'x';
-            board.GameBoard[5] = 'o';
-            board.GameBoard[6] = 'o';
-            board.GameBoard[7] = 'x';
-            board.GameBoard[8] = 'x';
+            Helpers helper = new Helpers();
+            int[] positions = { 1, 2, 3, 4, 5, 6, 7, 8 };
+            char[] symbols = { 'o', 'x', 'x', 'x', 'o', 'o', 'x', 'x' };
+            helper.FillBoard(board, positions, symbols);
             bool actual = board.CheckDraw('x', 'o');
             Assert.False(actual);
         }
+
+        [Fact]
+        public void ClearGameBoard()
+        {
+            Board board = new Board(3);
+            Helpers helper = new Helpers();
+            int[] positions = { 0, 1, 2, 3, 4, 5, 6, 7, 8 };
+            char[] symbols = { 'o', 'o', 'x', 'x', 'x', 'o', 'o', 'x', 'x' };
+            helper.FillBoard(board, positions, symbols);
+            board.InitializeBoard();
+            char[] actual = { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' };
+
+            Assert.Equal(board.GameBoard, actual);
```

Comment:
in this case, `actual` is actually what you expect.
and the assertion looks reversed as well, it should be
```
char[] expected = { ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ' };
Assert.Equal(expected, board.GameBoard);
```